templates/new_service/terraform/application/output.tf

```diff
@@ -1,3 +1,11 @@
 output "url" {
-  value = module.web_application.url
+  value = [
```

Comment:
Does this need to be a list?

Reply:
currently yes https://github.com/DFE-Digital/github-actions/blob/afaa2293e6215354bcaded426b15744a6b164a7f/deploy-to-aks/action.yml#L108

Reply:
changed url